history/conf.py

```diff
@@ -26,19 +26,24 @@
 kafka_group_id = os.environ.get('KAFKA_GROUP_ID', "history")
 
 # Global subject to use when publishing tenancy lifecycle events
-dojot_subject_tenancy = os.environ.get("DOJOT_SUBJECT_TENANCY", "dojot.tenancy")
+dojot_subject_tenancy = os.environ.get(
+    "DOJOT_SUBJECT_TENANCY", "dojot.tenancy")
 
 # Global subject to use when listening to device CRUD events
-dojot_subject_device = os.environ.get('DOJOT_SUBJECT_DEVICES', "dojot.device-manager.device")
+dojot_subject_device = os.environ.get(
+    'DOJOT_SUBJECT_DEVICES', "dojot.device-manager.device")
 
 # Global subject to use when listening to device data events
-dojot_subject_device_data = os.environ.get('DOJOT_SUBJECT_DEVICE_DATA', "device-data")
+dojot_subject_device_data = os.environ.get(
+    'DOJOT_SUBJECT_DEVICE_DATA', "device-data")
 
 # Global service to use when publishing dojot management events
 # such as new tenants
 dojot_service_management = os.environ.get("DOJOT_SERVICE_MANAGEMENT",
                                           "dojot-management")
 
+dojot_notification_on = os.environ.get("DOJOT_NOTIFICATION_ON", True)
```

Comment:
I think the name should be different, something like NOTIFICATION_ONLY

Reply:
Add this new env variable at README.md

Reply:
all right, the name of the variable will be changed.
And I will also add it to Readme.md